### ISSUE TITLE
{parents} tag pair not honouring {parents:title} variable (5.2.3)

### ISSUE THREAD

**Issue by directordash** [ExpressionEngine/ExpressionEngine#227]
```
{parents channel="news" limit="3"}
	<article class="sub_news">
		<h3><a href="/news/{parents:url_title}">{parents:title}</a></h3>
		 <h5 class="date mb-10">{parents:entry_date format="%d %M %Y"}</h5>
		 <p>{parents:body strip_tags="true" word_limit="16"}... <a href="/news/{parents:url_title}">read on</a></p>
	</article>
{/parents}
```

It's as simple as this:  all variables except for parents:title and parents:url_title are parsing correctly.  Even the parents:entry_id variable _does_ parse correctly (not shown here) so it's doing *something*

What I get is the url_title and title of a parent entry in another channel, repeated 3 times, with the entry_date, entry_id, body, etc of the entry I want to see.

**Comment by directordash** [comment]
UPDATE: if there's more than one iteration of {parents} within a channel entries loop, it's the second/third/et al loops that have this problem.
Closing the {exp:channel:entries} tag pair and reopening before the second iteration has helped, but an unnecessary load on the server.

**Comment by aasun** [comment]
This is the same problem as outlined in issue 221.  I tested the possible fix for my site and it has corrected the issue.

**Comment by robinsowell** [comment]
Thanks assun. It was the same issue and fixed in the 5.2.4 release.
